### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by woolfg**
- UI and message improvements
- button to create email including credentials

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+SHELL := /bin/bash
+.DEFAULT_GOAL := help
+
+.PHONY: help
+help: ## help message, list all command
```
Comment (mre) [inline Makefile]:
```suggestion
help: ## help message, list all commands
```

File: src/manifest.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
   "content_scripts": [
     {
       "matches": [
-        "*://*.openpodcast.dev/*"
+        "*://*.openpodcast.dev/*",
+        "*://*.openpodcast.app/*"
```
Comment (mre) [inline src/manifest.json:19]: 🎉

**Review by mre — APPROVED** [review]

**Comment by mre** [comment]
smart idea to send an email

**Comment by woolfg** [comment]
sumitted to chrome store, version 1.2

**Pull request merged**